insights/specs/default.py

```diff
@@ -672,6 +672,20 @@ def rhev_data_center(broker):
             return DatasourceProvider(content=json.dumps(bad_apples), relative_path=relative_path)
         raise SkipComponent()
 
+    @datasource(HostContext)
+    def ownership_of_var_opt_mssql(broker):
+        """Return the ownership of /var/opt/mssql if it is not mssql:mssql.
+        """
+        import json
+        root = broker[HostContext].root
+        relative_path = "var/opt/mssql"
+        path = os.path.join(root, relative_path)
+        if os.path.isdir(path):
+            name, group = get_owner(path)
+            if (name, group) != ("mssql", "mssql"):
```

Comment:
Do not add logic here.  just return name/group in datasource.
Logic is rule thing.